desktop/mac/permission-requirements.md

```diff
@@ -27,16 +97,6 @@ This approach has the following limitations:
 - Binding privileged ports (<1024) on `127.0.0.1` will not work. For example, `docker run -p 127.0.0.1:80:80 docker/getting-started` will fail, `docker run -p 80:80 docker/getting-started` however will succeed as binding privileged ports on `0.0.0.0` is no longer a privileged operation on recent versions of MacOS.
```

Comment:
remove this bullet point, as binding privileged ports can still be done if the user authorises the privileged service. Besides, this is already covered in the "Binding privileged ports" section